GameServer/Logic/Packets/Packets.cs

```diff
@@ -2435,131 +2435,30 @@ public class PlayerInfo : BasePacket
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xA9);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xC5);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xD7);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xD7);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0xD7);
-            buffer.Write((byte)0x14);
-            buffer.Write((byte)0x00);
-            buffer.Write((byte)0x00);
-
+            foreach (var runeCategory in player.GetChampion().Runes)
+            {
+                int required = 9;
+                if(runeCategory.Key.ToLower() == "black")
+                {
+                    required = 3;
+                }
+                for(int i = 1; i <= required; i++)
+                {
+                    short runeId = 0;
+                    if (runeCategory.Value.Count() >= i )
+                    {
+                        runeId = (short)runeCategory.Value[i - 1];
+                    }
+                    buffer.Write(runeId);
+                    buffer.Write((short)0x00);
+                }
+            }
+            
             var summonerSpells = player.getSummoners();
             buffer.Write((int)summonerSpells[0]);
-            buffer.Write((int)summonerSpells[1]);
+            buffer.Write((int)summonerSpells[1]);
+
+            #region masteries
```

Comment:
That region was already there but named "#region wtf" and I just moved it down and renamed it to clarify where masteries are defined. Will be removed when Masteries are done.